org.civicrm.reqangular/src/common/decorators/ui-bootstrap/uib-modal.decorator.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-env amd */
+
+define([], function () {
+  'use strict';
+
+  return ['$delegate', '$document', function ($delegate, $document) {
+    var openFunction = $delegate.open;
+
+    $delegate.open = open;
+
+    /**
+     * Decorates uibModal to set prevent body from scrolling while a modal is open
+     * and revert to the original styles values when it is closed
+     *
+     * @param  {any}
+     * @return {Object} Modal Instance
+     */
+    function open () {
+      var lockScrollStyling = ';overflow: hidden;height: 100%;width: 100%;';
+      var modalInstance;
+      var elements = [
+        { node: $document[0].body },
+        { node: $document[0].getElementsByTagName('html')[0] }
```

Comment:
you sure you need the html tag as well? Doesn't bootstrap deal with the body only?

Reply:
Exactly, Bootstrap only deals with `<body>`. As per https://stackoverflow.com/questions/3047337/does-overflowhidden-applied-to-body-work-on-iphone-safari (please see the answer with 57 upvotes). The `position: fixed` is an overkill though.